### ISSUE TITLE
Root certificate authority requires escape characters on windows OS

### ISSUE THREAD

**Issue by sk593** [radius-project/radius#6946]
### Steps to reproduce (required)

On Windows, follow this docs page on installing Radius with your own root certificate authority certificate: https://docs.radapp.io/guides/operations/kubernetes/install/#use-your-own-root-certificate-authority-certificate

### Observed behavior (required)

When the `rad install` command is run with a rootCA, an error is returned that the file path for the certificate cannot be found 
```
PS C:\WINDOWS\system32> rad install kubernetes --set-file global.rootCA.cert="C:\Users\Desktop\test.crt"
Installing Radius version v0.28.0 to namespace: radius-system...
Error: failed to add Radius values, err: open C:UsersDesktoptest.crt: The system cannot find the file specified., Helm output:
 
TraceId:  a9319a4e612397470986b0b125d574a6`
 
PS C:\WINDOWS\system32> rad install kubernetes --set-file global.rootCA.cert="C:\\Users\\Desktop\\test.crt"
Installing Radius version v0.28.0 to namespace: radius-system...
PS C:\WINDOWS\system32>
```

### Desired behavior (required)

We'd like this flag to succeed in setting the cert on Windows

Some options for fixing this issue: 
1. We handle the escape characters in the file path when the OS is Windows
2. We update our documentation to note that escape characters may be needed when providing the file path on Windows 

### Workaround (optional)

Using escape characters on the file path resolves the issue 
```
PS C:\WINDOWS\system32> rad install kubernetes --set-file global.rootCA.cert="C:\\Users\\Desktop\\test.crt"
Installing Radius version v0.28.0 to namespace: radius-system...
PS C:\WINDOWS\system32>
```

### rad Version (required)

RELEASE   VERSION   BICEP     COMMIT
0.28.0    v0.28.0   0.28.0    c7217b6f661a790053c1b65ff7972a0c5be33908

### Operating system (required)

Windows OS 

## Additional context

[AB#10750](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/10750)

**Comment by gpltaylor** [comment]
@rynowak can this be closed now?

**Comment by rynowak** [comment]
Yes, thanks for the reminder. This was fixed by #7479
